src/components/NewCommentForm.tsx

```diff
@@ -13,25 +40,33 @@ export const NewCommentForm: React.FC = () => {
             type="text"
             name="name"
             id="comment-author-name"
+            value={newComment?.name}
             placeholder="Name Surname"
-            className="input is-danger"
+            className={classNames('input', { 'is-danger': name })}
+            onChange={(event) => {
+              onCommentData(event, newComment);
```

Comment:
Pass only the 'event' to the onCommentData function. Remove the second argument 'newComment'.